templates/travis/.travis/assets/nginx.conf

```diff
@@ -41,17 +41,14 @@ http {
         # Gunicorn docs suggest this value.
         keepalive_timeout 5;
 
-        # Path to Pulp's static files.
-        root /usr/local/lib/pulp/lib/python3.7/site-packages/rest_framework;
-
         location /pulp/content/ {
             proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
             proxy_set_header X-Forwarded-Proto $scheme;
             proxy_set_header Host $http_host;
             # we don't want nginx trying to do something clever with
             # redirects, we set the Host: header above already.
             proxy_redirect off;
-            proxy_pass http://127.0.0.1:24816;
+            proxy_pass http://pulp-content;
```

Comment:
Where does this change come from? I am not sure it fits the single container architecture.

Reply:
it comes from here:
https://github.com/pulp/plugin_template/blob/master/templates/travis/.travis/assets/nginx.conf#L25-L27
and this change was applied to some plugins:
- https://github.com/pulp/pulp_rpm/blob/master/.travis/assets/nginx.conf
- https://github.com/pulp/pulp_ansible/blob/master/.travis/assets/nginx.conf

Reply:
Thank you for clarifying. I did no see this. And nginx config always seems to be good for a surprise.